src/store/getters/state-getters.ts

```diff
@@ -115,3 +115,55 @@ export const isBusySaving = (state: any): boolean => {
 export const isApplicationValid = (state: any): boolean => {
   return (state.stateModel.defineCompanyStep.valid && state.stateModel.addPeopleAndRoleStep.valid)
 }
+
+/**
+ * Steps to be displayed in the stepper
+ */
+export const getSteps = (state: any, getters:any): Array<any> => {
+  const steps: Array<any> = [{
+    id: 'step-1-btn',
+    step: 1,
+    icon: 'mdi-domain',
+    text: 'Define Your\nCompany',
+    to: '/define-company',
+    disabled: getters.isBusySaving,
+    valid: state.stateModel.defineCompanyStep.valid
```

Comment:
Not sure if it is overkill, but you could* make a steps Interface and apply that type definition to the `const steps: Array<StepIF>`.  As well as the the return on the method `getSteps`.